system/integrations/ldap/index.rst

```diff
@@ -0,0 +1,94 @@
+LDAP / Active Directory
+=======================
+
+Zammad comes with a powerful LDAP integration that allows you to have a single
+source of truth. By this you reduce the number of login credentials your user
+has to remember.
+
+.. hint::
+
+   The LDAP source is also a perfect candidate for Zammads
+   `Kerberos Single Sign-On`_ but also works as addition to other
+   :doc:`/settings/security/third-party`.
+
+.. _Kerberos Single Sign-On:
+   https://docs.zammad.org/en/latest/appendix/single-sign-on.html
+
+.. figure:: /images/system/integrations/ldap/ldap-integration-management.png
+   :width: 90%
+   :align: center
+   :alt: Zammands LDAP management page
+
+.. toctree::
+   :hidden:
+
+   manage
+
+📝 :doc:`manage`
+   Add, modify, remove or re-arrange your LDAP-Sources as needed.
+
+Limitations
+-----------
+
+Before you continue, please note the following limitations.
+
+   * Mapping / Synchronizing organizations is not possible
+
+        .. tip::
+
+           You may want to consider using domain based assignments to overcome
+           this issue. Learn more on :doc:`/manage/organizations/index`.
+
+   * Zammads LDAP sync is one way. Editing user settings or permissions may be
+     overwritten upon the next sync depending on your configuration.
+   * Nested groups are not supported.
+   * Synchronizing user avatars from LDAP is not supported.
+   * Unlike user filters, group filters cannot be changed.
+   * When a user originates from an LDAP server, Zammad will try to verify the
+     login credentials against LDAP first - if this fails Zammad will check its
+     local database.
+
+        .. warning::
+
+           Users can have local passwords even if they're LDAP users!
+           You can learn more about user accounts in general on
+           :doc:`/manage/users/index`.
+
+   * When using several LDAP sources containing users with email addresses
+     on different sources, the user in question will be updated with every
```

Comment:
When several LDAP sources contain the same user (meaning the same email address), the user in question ...